app/controllers/stories_controller.rb

```diff
@@ -202,6 +202,7 @@ def assign_article_show_variables
     assign_second_and_third_user
     not_found if permission_denied?
     @comment = Comment.new(body_markdown: @article&.comment_template)
+    @suggested_articles = ArticleSuggester.new(@article).articles(max: 4)
```

Comment:
Mmm I think you're rightm this way I always load the suggested article, instead of being loaded every 18 hours here: 

https://github.com/thepracticaldev/dev.to/blob/166e26bdf9c7c79bfd5a24839cf03dc861eef97b/app/views/articles/show.html.erb#L199-L200

I'll move it down again. Great catch and thanks for the review

Reply:
@lightalloy done :)